recipes/rsem/build.sh

```diff
@@ -34,6 +37,55 @@ rsem-tbam2gbam \
 rsem_perl_utils.pm \
 "
 
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+
+########################################################
+### Prepare Samtools/Htslib 
+########################################################
+
+sed -i.bak 's/^CPPFLAGS =$//g' samtools-1.3/Makefile
```

Comment:
RSEM needs to compile against specific samtools libraries, and for my first iteration I just wanted to get it working. I'm going to rework the `Makefile` it so I don't have to do this.